### DISCUSSION TITLE
Trying to use Apple music through Alexa media player group in home assistant

### DISCUSSION THREAD

**Discussion by tismofied** [music-assistant/support#3173] [Q&A] [answered]
I keep getting this prompt `
to send tts, set url in integration configuration `
whenever I press play.
Where do I find more info on this please?
I made a little YouTube short to understand better

[trying to play song through ha music assistant ](https://youtube.com/shorts/6bc1eX_-L24?si=x_A3-f54ZEVrr3nG)

Don't mind the song. I just picked up the first song that came up in Apple music search

**Comment by OzGav** [comment, marked as answer]
Alexa is not supported https://github.com/orgs/music-assistant/discussions/431
